examples/raspicam/README.md

```diff
@@ -1,5 +1,48 @@
-# How to compile    
-- g++    
+# How to compile  
+
+### Installing Dependencies
+- [RaspiCam](https://www.uco.es/investiga/grupos/ava/node/40)
+	+ Download the RaspiCam archive from [here](https://sourceforge.net/projects/raspicam/files/latest/download)
+	+ Extract the archive
+		``` 
+			# if .tar archive
+			tar xvzf raspicam-xxxxxx.tar 
+			# or if .zip archive
+			unzip raspicam-xxxxxx.zip
+		```
+	+ cd to the extracted folder
+		``` 
+		cd raspicam-xxxxxx
+		```
+	+ create a buid dir and start compiling
+		```  
+		mkdir build
+		cd build
+		cmake ..
+		```
+	+  At this point you'll see something like 
+	```
+	-- CREATE OPENCV MODULE=1
```

Comment:
How does OpenCV relate to raspicam?

Reply:
I added the OpenCV support compilation step as people might use the raspicam library for other projects too.
Will remove that section.

Reply:
Done. Please review.